test/Broadway/EventSourcing/AbstractEventSourcingRepositoryTest.php

```diff
@@ -146,7 +146,7 @@ public function it_calls_the_event_stream_decorators_with_the_correct_arguments(
         $lastCall = $this->eventStreamDecorator->getLastCall();
 
         $this->assertEquals($aggregate->getAggregateRootId(), $lastCall['aggregateIdentifier']);
-        $this->assertEquals('\\' . get_class($aggregate), $lastCall['aggregateType']);
+        $this->assertEquals(get_class($aggregate), $this->stripLeadingSlash($lastCall['aggregateType']));
```

Comment:
Would you mind adding a test for both situations for sake of regression testing?

Reply:
I don't understand how, since this already is a test

Reply:
You mentioned it breaks when creating a repository using aggregate::class notation and not when using the string FQDN representation. I would like to see a test for both conditions. 

But I've tried it manually by modifying https://github.com/broadway/broadway/blob/master/test/Broadway/EventSourcing/EventSourcingRepositoryTest.php#L29 and your change works, so I'm ok with it.